tests/complexity/scripts/ex1.py

```diff
@@ -0,0 +1,355 @@
+# ...
```

Comment:
Usually we don't insist on documentation in test files. So all new files created in the `tests/` folder should contain the following in the first line of the file:
```
# pylint: disable=missing-function-docstring, missing-module-docstring
```